roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -79,7 +79,11 @@ reserved_host_memory_mb = {{ nova.compute_reserve_ram }}
 reserved_host_disk_mb = {{ nova.reserved_host_disk_mb }}
 
 # Misc #
+{% if ironic.enabled -%}
+compute_driver = ironic.IronicDriver
+{% else %}
```

Comment:
Yeah, we'll want to make more adjustments to this file to make good on the story of being able to run both VM and bare metal at the same time.